tools/linker/MonoTouch.Tuner/RemoveBitcodeIncompatibleCodeStep.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright 2012-2014, 2016 Xamarin Inc. All rights reserved.
+using System.Collections.Generic;
+using Mono.Tuner;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Mono.Linker.Steps;
+
+using Xamarin.Bundler;
+using Xamarin.Linker;
+using Mono.Linker;
+using System;
+
+using Xamarin.Tuner;
+using Xamarin.Linker;
+
+namespace MonoTouch.Tuner {
+	public class RemoveBitcodeIncompatibleCodeStep : BaseSubStep {
```

Comment:
oops, I guess I should have fully completed my train of thoughts... :)
`ExceptionalSubStep` makes support a lot easier if an exception is thrown during processing (which was the main drawback of substeps), otherwise there's no way to know (from logs) what caused the issue